src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+const App = () => (
+  <div id="app">
+    <Display />
+    <ButtonPanel />
+  </div>
+);
```

Comment:
- [ ] Rather than using a div to wrap display and buttonpanel components use React.Fragment to avoid adding an unnecessary div node.

Reply:
I used the shorthand <></> because eslint did not allow me to use React.Fragment